packages/prerender-fargate/README.md

```diff
@@ -1,18 +1,113 @@
-# Prerender in Fargate
-
-A construct to host [Prerender](https://github.com/prerender/prerender) in Fargate.
-
-## Props
-
-- `prerenderName`: Name of the Prerender service
-- `domainName`: Domain name for Prerender
-- `vpcId`: VPC to host Prerender in
-- `bucketName`: Optional S3 bucket name
-- `expirationDays`: Optional days until items expire in bucket (default to 7 days)
-- `tokenList`: List of tokens to accept as authentication
-- `certificateArn`: Certificate arn to match the domain
-- `desiredInstanceCount`: Number of Prerender instances to run (default 1)
-- `maxInstanceCount`: Maximum number of Prerender instances to run (default 2)
-- `instanceCPU`: CPU to allocate to each instance (default 512)
-- `instanceMemory`: Amount of memory to allocate to each instance (default 1024)
-- `enableRedirectCache`: Cache 301 and 302 responses, too (default false)
+# PrerenderFargate Construct
+
+The PrerenderFargate construct sets up an AWS Fargate service to run a [Prerender] service in an ECS Fargate cluster.  
```

Comment:
Need to add the link to prerender.io :sweat_smile:

Reply:
I have included in the Acknowledgments section